CMakeLists.txt

```diff
@@ -0,0 +1,130 @@
+project(zbar_qrcode)
+cmake_minimum_required(VERSION 3.10)
+
+set(CMAKE_C_STANDARD 99)
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
+
+option(BUILD_SHARED_LIBS "Build shared library" ON)
+
+set(CMAKE_THREAD_PREFER_PTHREAD ON)
```

Comment:
This is for windows only?

[very minor/style] It may make sense to put under ifs code that only applies to one platform (if it does). That way later on we can know which platforms are affected by changing or removing it.